Dockerfile

```diff
@@ -1,22 +1,41 @@
-FROM ubuntu:22.04
```

Comment:
The official guidance from Milk is to use an Ubuntu base image. I am unsure if they have any requirements based on this, but this may cause a regression